### PR TITLE
Improving and extending HiCExplorer tutorial

### PR DESCRIPTION
**Pull request by joachimwolff**
- Adding Pearson matrices and PCA tracks
- Adding loop detection

### CODE REVIEW

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 1]
```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
...
+This dataset is from the human cell GM12878, mapped to hg19 and of 10 kb resolution. We use a new file because to detect loop structures the read coverage is required to be in the hunderts of million; this was not the case for the previous used drosophila dataset.
+
+> ### {% icon hands_on %} Hands-on: Matrix information 
+>
+> 1. **hicInfo** {% icon tool %}: Run hicInfo adjusting the parameters:
+>    - "Select" `Multiple datasets`
+>    - "Matrix to compute on" to `corrected contact matrix dm3 large` and `GM12878-MboI-allreps-filtered.10kb.cool`
+{: .hands_on}
+
+We now investigate the result of hicInfo and see that the new importet file is having 1.2 billion non-zero elements, while the drosophila Hi-C interaction matrix has around 12 million non-zero elements.
```
Comment (mtekman) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: importet → imported

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 2]
```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
...
+>    - "Peak width" to `6`
+>    - "Window size" to `10`
+>    - "P-value preselection" to `0.01`
+>    - "P-value" to `0.01`
+>    - "Chromosomes to include" to `chr1`
+>
+{: .hands_on}
+
+
+The detection of the loops is based on a preselection of interactions, per interaction a p-value given a continuous negative binomial distribution over all interactions of a relative distance is computed. In a second step, the selected peak candidate is compared against its background using a Wilcoxon rank-sum test.
```
Comment (anuprulez) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: `the pre-selection of interactions ..., p-value is computed per interaction given a continuous negative ...`

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 3]
```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
 
 The resulting image should look like this one:
 ![TAD plot](../../images/plotTADs.png)
+
+# Loop detection
+
+In Hi-C data, the term `loop` refers to a 3D structure which represent enhancer-promoter, gene, architectural or polycomb-mediated interactions. These interactions have the characteristics to be enriched in a single region compared to the local background. These loops are also so called long-range interactions with an expected maximum distance of 2 MB (see [Rao et al. 2014](https://doi.org/10.1016/j.cell.2014.11.021).
```
Comment (anuprulez) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: `which represents enhancer-promoter ... `

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 4]
```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
...
+>
+{: .hands_on}
+
+![Loops result_plot](../../images/loops_plot.png)
+
 # Conclusion
 {:.no_toc}
 
-In this tutorial we used HiCExplorer to analyze drosophila melanogaster cells. We mapped the chimeric reads and created a contact matrix, to reduce noise this contact matrix was normalized. We showed how to visualize a contact matrix and how we can investigate topological associating domains and relate them to additional data like gene tracks.
+In this tutorial we used HiCExplorer to analyze drosophila melanogaster cells. We mapped the chimeric reads and created a contact matrix, to reduce noise this contact matrix was normalized. We showed how to visualize a contact matrix and how we can investigate topological associating domains and relate them to additional data like gene tracks. Moreover, we used a human Hi-C interaction matrix to compute loop structures.
```
Comment (anuprulez) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: `We mapped chimeric reads ...`

**Comment by joachimwolff** [comment]
ping @bgruening Needs to be merged for the Thursday Hi-C Galaxy course.

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 5]
```diff
@@ -257,18 +261,48 @@ TAD calling works in two steps: First HiCExplorer computes a TAD-separation scor
...
+
+
+![Pearson PC2](../../images/pearson_pc2.png)
+
+The first principal component correlates with the chromosome arms, while the second component correlates with A/B compartments. 
+
 # Integrating Hi-C and other data
 
-We can plot the TADs for a given chromosomal region. For this we will use [hicPlotTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html).
+We can plot the TADs for a given chromosomal region. For this we will use [pyGenomeTracks](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html).
```
Comment (bgruening) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: the link should point to pyGenomeTracks?

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md [hunk 6]
```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
 
 The resulting image should look like this one:
 ![TAD plot](../../images/plotTADs.png)
+
+# Loop detection
+
+In Hi-C data, the term `loop` refers to a 3D structure which represents enhancer-promoter, gene, architectural or polycomb-mediated interactions. These interactions have the characteristics to be enriched in a single region compared to the local background. These loops are also called long-range interactions with an expected maximum distance of 2 MB (see [Rao et al. 2014](https://doi.org/10.1016/j.cell.2014.11.021).
```
Comment (bgruening) [inline topics/epigenetics/tutorials/hicexplorer/tutorial.md]: missing bracket in the end

File: topics/epigenetics/tutorials/hicexplorer/tutorial.md (hunk 5 above)
Comment (joachimwolff) [reply topics/epigenetics/tutorials/hicexplorer/tutorial.md]: No, in the documentation of HiCExplorer it is still hicPlotTads, and the readthedocs of pgt is not public at the moment.

**Comment by joachimwolff** [comment]
@bgruening Done.

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**